tools/vast-repl/command.cpp

```diff
@@ -108,11 +109,26 @@ namespace vast::repl::cmd {
     void meta::run(state_t &state) const {
         check_and_emit_module(state);
 
-        auto action  = get_param< action_param >(params);
+        auto action = get_param< action_param >(params);
         switch (action) {
-            case meta_action::add: add(state); break;
-            case meta_action::get: get(state); break;
+            case meta_action::add:
+                add(state);
+                break;
+            case meta_action::get:
+                get(state);
+                break;
         }
-    };
+    }
+
+    //
+    // apply command
+    //
+    void apply::run(state_t &state) const {
+        check_and_emit_module(state);
+
+        auto [tm, th] = tower::default_manager_t::get(state.ctx, std::move(state.mod));
+
+        llvm::outs() << tm.apply(th, createHLToLLCFPass()).mod << "\n";
```

Comment:
This replicates functionality of `show module`.

If this is supposed to create new layer of tower lets call it `raise` and we can keep tower in the state.

In fact my idea was that `vast_module` will eventually be replaced with `tower`.
It  shloud be irrelevant whether `tower` has single layer or more.